Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -69,16 +70,20 @@ public class BTShopperInsightsClient {
                 isVenmoRecommended: venmo?.recommended ?? false,
                 isEligibleInPayPalNetwork: payPal?.eligibleInPayPalNetwork ?? false || venmo?.eligibleInPayPalNetwork ?? false
             )
-            return self.notifySuccess(with: result)
+            return self.notifySuccess(with: result, for: experiment)
         } catch {
-            throw self.notifyFailure(with: error)
+            throw self.notifyFailure(with: error, for: experiment)
         }
     }
 
     /// Call this method when the PayPal button has been successfully displayed to the buyer.
     /// This method sends analytics to help improve the Shopper Insights feature experience.
-    public func sendPayPalPresentedEvent() {
-        apiClient.sendAnalyticsEvent(BTShopperInsightsAnalytics.payPalPresented)
+    public func sendPayPalPresentedEvent(experiment: String? = nil, rank: Int? = nil) {
```

Comment:
I don't think `rank` will mean much to merchants so maybe keeping the external facing name as button rank or something similar will be better. We should also update the docstrings for this method.